Unhandled exception in the UI thread.

```
GAPID Version: 0.6.1:0:developer
OS: linux amd64

java.lang.reflect.UndeclaredThrowableException
	at com.sun.proxy.$Proxy0.onMemoryFollowed(Unknown Source)
	at com.google.gapid.models.Follower.onFollow(Follower.java:207)
	at com.google.gapid.views.AtomTree$2.mouseDown(AtomTree.java:231)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(Unknown Source)
	at org.eclipse.swt.widgets.EventTable.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Display.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Widget.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Unknown Source)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Unknown Source)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:818)
	at org.eclipse.jface.window.Window.open(Window.java:794)
	at com.google.gapid.Main$UI.show(Main.java:109)
	at com.google.gapid.Main.main(Main.java:67)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.gapid.util.Events$ListenerCollectionImpl.invoke(Events.java:157)
	... 13 more
Caused by: java.lang.NullPointerException
	at com.google.gapid.views.MemoryView$Selections.updateSelectedObservation(MemoryView.java:346)
	at com.google.gapid.views.MemoryView.update(MemoryView.java:259)
	at com.google.gapid.views.MemoryView.onMemoryFollowed(MemoryView.java:231)
	... 18 more
```

--- a/gapic/src/main/com/google/gapid/views/MemoryView.java
+++ b/gapic/src/main/com/google/gapid/views/MemoryView.java
@@ -26,6 +26,7 @@ import static com.google.gapid.widgets.Widgets.createDropDownViewer;
 import static com.google.gapid.widgets.Widgets.createLabel;
 import static com.google.gapid.widgets.Widgets.ifNotDisposed;
 import static com.google.gapid.widgets.Widgets.scheduleIfNotDisposed;
+import static java.util.Collections.emptyList;
 import static java.util.logging.Level.WARNING;
 
 import com.google.common.collect.Lists;
@@ -322,6 +323,7 @@ public class MemoryView extends Composite
       combo.setContentProvider(ArrayContentProvider.getInstance());
       combo.setLabelProvider(new LabelProvider());
       combo.getCombo().setLayoutData(new GridData(SWT.FILL, SWT.CENTER, false, false));
+      combo.setInput(emptyList());
       return combo;
     }
 
@@ -331,7 +333,7 @@ public class MemoryView extends Composite
 
     public void setObservations(Observation[] observations) {
       if (observations.length == 0) {
-        obsCombo.setInput(Arrays.asList(observations));
+        obsCombo.setInput(emptyList());
       } else {
         obsCombo.setInput(Lists.asList(Observation.NULL_OBSERVATION, observations));
       }

